Lib/test/test_warnings/__init__.py

```diff
@@ -1684,6 +1684,31 @@ def d(): pass
             isinstance(cell.cell_contents, deprecated) for cell in d.__closure__
         ))
 
+    def test_inspect(self):
+        import inspect
```

Comment:
nit: I'd probably just move this import to the top of `Lib/test/test_warnings/__init__.py`. But it doesn't make too much difference either way.